build.gradle

```diff
@@ -3,13 +3,13 @@ plugins {
   id 'checkstyle'
   id 'idea'
   id 'jacoco'
+  id 'se.patrikerdes.use-latest-versions' version '0.2.18'
   id 'com.github.ben-manes.versions' version '0.51.0'
-  id 'org.owasp.dependencycheck' version '9.0.9'
-  id 'io.freefair.lombok' version '6.5.1'
-  id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-  id 'org.sonarqube' version '3.3'
-  id 'org.springframework.boot' version '2.7.18'
-  id 'uk.gov.hmcts.java' version '0.12.40'
+  id 'org.owasp.dependencycheck' version '11.0.0'
+  id 'io.spring.dependency-management' version '1.1.5'
+  id 'org.sonarqube' version '5.1.0.4882'
+  id 'org.springframework.boot' version '3.3.5'
+  id 'uk.gov.hmcts.java' version '0.12.57'
```

Comment:
This, I believe, could also use later version, '0.12.63', unless there is a specific reason not to

Reply:
will upgrade all dependency versions to latest once closer to releasing